scripts/install-mender.sh

```diff
@@ -466,7 +466,14 @@ do_setup_mender_client() {
 
     echo "  Setting up mender with options: $MENDER_SETUP_ARGS"
     `mender_setup_cli` $MENDER_SETUP_ARGS
-    pidof systemd && systemctl restart mender-client
+    pidof systemd && (
+        if [[ "$SELECTED_COMPONENTS" == *"mender-client4"* ]]; then
+            systemctl restart mender-authd
+            systemctl restart mender-updated
+        else
+            systemctl restart mender-client
+        fi
+    )
```

Comment:
```suggestion
    pidof systemd && {
        if [[ "$SELECTED_COMPONENTS" == *"mender-client4"* ]]; then
            systemctl restart mender-authd
            systemctl restart mender-updated
        else
            systemctl restart mender-client
        fi
    }
```
no need for subshell here. (unless it was on purpose)

Reply:
thanks!